templates/d1/wrangler.toml

```diff
@@ -0,0 +1,24 @@
+name = "honc-d1-template"
+compatibility_date = "2024-07-25"
+compatibility_flags = [ "nodejs_compat" ]
+
+[[d1_databases]]
+binding = "DB"
+database_name = "d1-database"
+database_id = "32de90b4-702c-4ddf-80d1-000b06acd51e"
```

Comment:
is it a good idea to commit the `id` and `name` of the database? It seems like it would be something that the user adds after running `wrangler d1 create ...` etc